### DISCUSSION TITLE
Adding another inidcator as a cusom button

### DISCUSSION THREAD

**Discussion by cubecleveland** [JedWatson/react-select#4925] [Q&A] [answered]
Hello, im trying to add 'another indicator to the dropdown select which will behave as an independent button  with its own click handler.

I was trying to stopPropagation() on the button click but it still opens the menu whenever a click is preformed whitihng the indicator section. is there any way to override it ? or shoul i add it under another compoenent. here si what im trying to achive

![image](https://user-images.githubusercontent.com/51701990/142772450-3305090d-fd42-45ef-9c83-11a6d60459de.png)

**Comment by ebonow** [comment, marked as answer]
@cubecleveland there are a few events that need to be prevented from propagating to the Control component.

Specifically, `onMouseDown`, `onTouchEnd`, and `onKeyDown`.

Here is an example codesandbox that should work for your use-case.

https://codesandbox.io/s/react-select-add-new-indicator-v9eox
